ethpm/validation.py

```diff
@@ -17,6 +19,16 @@ def validate_package_name(pkg_name: str) -> None:
         raise UriNotSupportedError("{0} is not a valid package name.".format(pkg_name))
 
 
+def validate_build_dependencies(dependencies: Dict[str, str]) -> None:
+    """
+    Raise an exception if the key in dependencies is not a valid package name,
+    or if the value is not a valid IPFS URI.
+    """
+    for key, value in dependencies.items():
+        validate_package_name(key)
+        is_ipfs_uri(value)
```

Comment:
This function just returns `True/False`.  I'd suggest creating a new `validate_ipfs_URI` function:

```python
def validate_ipfs_URI(value):
    if not is_ipfs_uri(value):
        raise ValidationError(...)
```